policyengine_us/reforms/tax_employer_payroll/tax_employer_payroll.py

```diff
@@ -0,0 +1,131 @@
+from policyengine_us.model_api import *
+from policyengine_core.periods import period as period_
+
+
+def tax_employer_payroll_reform() -> Reform:
+    """
+    General workflow for creating reform:
+    1. Make a boolean for the reform under parameters/contrib/{reform_name}
+    called in_effect.
+    2. Create new variables if necessary.
+    3. Redefine rule for calculating existing variable (in this case social
+    security taxation).
+    4. Add this reform to reforms/reforms.py.
+    """
+
+    """
+    Create a variable with the same name as the variable we are trying to 
+    amend.
+    In this case it's "irs_gross_income", as in: 
+    "policyengine_us/variables/gov/irs/income/taxable_income/
+    adjusted_gross_income/irs_gross_income/irs_gross_income.py"
+    We amend irs_gross_income because the employer-side contributions 
+    will go into taxable income...
+    but taxable income is determined at the tax unit level, 
+    by summing up all persons' gross income 
+    and deducting tax unit's deductions. But since payroll is calculated at 
+    the personal level, 
+    we want to add each person's employer contribution to each person, thus 
+    we add it to gross income.
+    """
+
+    class irs_gross_income(Variable):
+        # The input is Variable because policyengine.core functions(?) will
+        # find the variables we name and feed it into this class
+
+        # The following are attributes copied from the .../irs_gross_income.py
+        value_type = float
+        entity = Person
+        label = "Gross income"
+        unit = USD
+        documentation = "Gross income as defined in the Internal Revenue Code."
+        definition_period = YEAR
+        reference = "https://www.law.cornell.edu/uscode/text/26/61"
+
+        # the formula is also copied from the .../irs_gross_income.py
+        def formula(person, period, parameters):
+            sources = parameters(period).gov.irs.gross_income.sources
```

Comment:
let's instead make this a parametric reform updating this list to include the sources